paddlets/ensemble/weighting_ensemble_forecaster.py

```diff
@@ -37,8 +37,8 @@ def __init__(self,
                  out_chunk_len: int,
                  skip_chunk_len: int,
                  estimators: List[Tuple[object, dict]],
-                 mode="mean",
-                 verbose: bool = False
+                 mode: Optional[str]="mean",
+                 verbose: Optional[bool] = False
```

Comment:
Line40 - Line41 mode, verbose should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional